From aa246fb9055bdcb293e9daca571066b72adc1cb2 Mon Sep 17 00:00:00 2001
From: Li Xin <lixin.fnst@cn.fujitsu.com>
Date: Thu, 14 Jan 2016 17:54:27 +0900
Subject: [PATCH] conf-ld: warning fix

WARNING: File 'usr/bin/chkshsgr' from daemontools-native was already stripped, this will prevent future debugging!

Upstream-Status: pending

Signed-off-by: Li Xin <lixin.fnst@cn.fujitsu.com>
---
 src/conf-ld | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/conf-ld b/src/conf-ld
index 59a0de7..1d0518a 100644
--- a/src/conf-ld
+++ b/src/conf-ld
@@ -1,3 +1,3 @@
-gcc -s
+gcc
 
 This will be used to link .o files into an executable.
-- 
1.8.4.2

